static/js/stat_var_hierarchy/stat_var_section_input.tsx

```diff
@@ -120,33 +142,69 @@ export class StatVarSectionInput extends React.Component<
     );
   }
 
-  private mouseMoveAction = (e) => {
-    let html = "This stat var has no data for any of the chosen places.";
-    if (!_.isEmpty(this.props.summary)) {
-      let availablePlaceTypes = Object.keys(
-        this.props.summary.placeTypeSummary
-      );
+  private getTooltipHtml(hasData: boolean): string {
+    let html = hasData
+      ? ""
+      : "This variable has no data for any of the chosen places.";
+    if (_.isEmpty(this.props.summary)) {
+      return html;
+    }
+    const availablePlaceTypes = Object.keys(
+      this.props.summary.placeTypeSummary
+    ).filter((placeType) => {
       if (this.context.statVarHierarchyType === StatVarHierarchyType.MAP) {
-        availablePlaceTypes = availablePlaceTypes.filter(
-          (placeType) =>
-            placeType in USA_CHILD_PLACE_TYPES && placeType !== "Country"
-        );
+        return placeType in USA_CHILD_PLACE_TYPES && placeType !== "Country";
       }
-      if (availablePlaceTypes.length === 0) {
-        html = "Sorry, this stat var is not supported by this tool.";
-      } else {
-        html += " You can try these types of places instead. <ul>";
+      return !IGNORED_PLACE_TYPES.has(placeType);
+    });
+    if (availablePlaceTypes.length === 0) {
+      html = "Sorry, this variable is not supported by this tool.";
```

Comment:
this case doesn't start the list. consider building the html in the following way:
- build a list of place types
- if list is non-empty, wrap in each with li, and the list with ul.

Reply:
done

Reply:
done